server/application/rest/work_shift.py

```diff
@@ -65,18 +66,25 @@ def work_shifts():
         qrystr_params = {
             "filters": {},
         }
-        for arg, values in request.args.items():
-            print(arg, values)
+        facility_id = request.args.get("code")
+        if facility_id:
+            response_from_external_api = requests.get(f"https://api2-qa.gethelp.com/v2/facilities/{facility_id}")
+            if response_from_external_api.status_code == 200:
+                get_facility_info = response_from_external_api.json()
+                qrystr_params['get_facility_info'] = get_facility_info
+            else:
+                print("Failed to retrieve facility info from external API")
```

Comment:
Please don't put this code here. Create a new use-case for retrieving facility info by id. The id of the facility comes from the return value of the workshift_list_use_case.